translations/OpenOrienteering_sv.ts

```diff
@@ -438,12 +438,12 @@
     <message>
         <location filename="../src/color_dock_widget.cpp" line="48"/>
         <source>CMYK</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Or looking into it it's not a mistake, sorry saw this from my phone.

The past translator did just leave the finished element empty when the
value should be equal to the source...